zenodo/modules/records/templates/zenodo_records/box/edit.html

```diff
@@ -72,36 +72,38 @@
       <span class="text-danger"><i class="fa fa-trash"></i> Delete user ...</span>
     </a></li>
     {%- endif %}
-    {% if record is safelisted_record %}
-    <li>
-      <form action="{{ url_for('zenodo_spam.safelist_add_remove',
-      user_id=record._deposit.owners[0]) }}" method="POST">
-        <input type="hidden" name="action" value="delete">
-         <input type="hidden" name="next"
-               value="{{ url_for('invenio_records_ui.recid',
-               pid_value=pid.pid_value) }}"
-        />
-        <button class="btn btn-link" type="submit" {{
-          'disabled' if edit_disabled else ''}}><div style="color: #f0ad4e">
-          <i class="fa fa-pencil-square-o"></i> Remove from safelist</div>
-        </button>
-      </form>
-    </li>
-    {% else %}
-    <li>
-      <form action="{{ url_for('zenodo_spam.safelist_add_remove',
-       user_id=record._deposit.owners[0]) }}" method="POST">
-        <input type="hidden" name="action" value="post">
-        <input type="hidden" name="next"
-               value="{{ url_for('invenio_records_ui.recid',
-               pid_value=pid.pid_value) }}"
-        />
-        <button class="btn btn-link" type="submit" {{
-          'disabled' if edit_disabled else ''}}><div style="color: #6aa3d5">
-          <i class="fa fa-pencil-square-o"></i> Add to safelist</div>
-        </button>
-      </form>
-    </li>
+    {% if record._deposit.owners and record._deposit.owners[0] %}
```

Comment:
Basically adding this line should fix this

Reply:
```suggestion
    {% set has_owners = record._deposit.owners and record._deposit.owners[0] %}
```

Maybe something like this, which you can re-use below to determine if you render the option.